content/en/docs/concepts/performance-and-scalability/index.md

```diff
@@ -101,20 +101,20 @@ immediately. This process adds to the queue wait time of the next request and af
 average and tail latencies. The actual tail latency depends on the traffic pattern.
 
 Inside the mesh, a request traverses the client-side proxy and then the server-side
-proxy. This two proxies on the data path add about 8ms to the 90th percentile latency at 1000 requests per second.
-The server-side proxy alone adds 2ms to the 90th percentile latency.
+proxy. This two proxies on the data path add about 7ms to the 90th percentile latency at 1000 requests per second.
+The server-side proxy alone adds 3ms to the 90th percentile latency.
```

Comment:
these numbers seem suspicious. Are you sure this is not standard variance? 7ms for end to end, where 3ms goes to server, leaving 4ms for the client including the network.
compared to previous one of 8ms end to end, where 2ms goes to server, leaving 6ms for the client including the network.

Reply:
Thanks for your comments, I need to confirm with @mandarjog about the testing data.

Reply:
@rshriram Thanks for your concern, checked again with the data, this serverside proxy alone should adds around 2ms. which is calculated by 3.1-0.82.